configs/components/rubygem-bindata.rb

```diff
@@ -0,0 +1,5 @@
+component "bindata" do |pkg, settings, platform|
```

Comment:
Though this will work (build successfully) with the component name `bindata` the convention in this repo has been to call the components by the same name as the file they are defined in. So total nit but I think the component names should be updated with prefix `rubygem` to match the convention in the rest of the repo. `rubygem-bindata`.

Reply:
Also we will typically mirror these gems in artifactory. I just went ahead and updated for all these with kermi. 
![image](https://user-images.githubusercontent.com/15354214/53436209-87af8d80-39af-11e9-9028-babf0af741bf.png)

Reply:
Ah great! I used to do this with stickler all the time, but haven't needed to since we moved to Artifactory.